dotCMS/src/main/java/com/dotcms/publisher/bundle/business/BundleAPI.java

```diff
@@ -168,4 +168,10 @@
 	 */
 	public void deleteAssetFromBundle(String assetId, String bundleId) throws DotDataException;
 
+    /**
+     * This takes a bundle and generates the tar.gzipped output file.  The resulting file will be placed under
+     * the ConfigUtils.getBundlePath() + "/" + bundleId + ".tar.gz"
+     */
+    public File generateTarGzipBundleFile(Bundle bundle);
```

Comment:
Codacy found an issue: [Unnecessary modifier 'public' on method 'generateTarGzipBundleFile': the method is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6239861)